.happy/config.json

```
@@ -3,7 +3,7 @@
     "terraform_version": "0.13.5",
     "default_env": "rdev",
     "app": "hosted-cellxgene",
-    "default_compose_env": ".env.ecr",
+    "default_compose_env_file": ".env.ecr",
```

```suggestion
    "default_compose_env_file": ".env.ecr",
    "default_compose_env": ".env.ecr",
```
I will keep both for now so that we don't break python

---

oh woops, maybe I should have been the one to accept this suggestion prior to approval? looks like it merged w/o it.